docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,84 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since libsodium's tests do not provide reference data for the SealedBox
+construction, the implementation is verified with a ``sealbox_test_vectors``
+utility program that produces and checks custom test vectors
+by making specific calls to libsodium API.
+
+To build the ``sealbox_test_vectors`` you need a ``C`` language compiler,
+a prebuilt libsodium library more recent than version 1.0.3 and the
+corresponding include headers.
+
+In a unix-like programming environment you shoud then execute:
+
+.. code-block:: bash
+
+    $ cc -o sealbox_test_vectors -l sodium sealbox_test_vectors.c
```

Comment:
The compiling errors I got were due to this command. It's either incompatible with `cc` bundled with my Ubuntu Xenial, or it's incorrect and should say 

    cc sealbox_test_vectors.c -lsodium -o sealbox_test_vectors